app/helpers/dashboard_helper.rb

```diff
@@ -1,16 +1,19 @@
 module DashboardHelper
-  def projects_dashboard_filter_path(options={})
-    exist_opts = {
-      sort: params[:sort],
-      scope: params[:scope],
-      group: params[:group],
-    }
+  def filter_path(entity, options={})
```

Comment:
Unused method argument - `entity`. If it's necessary, use `_` or `_entity` as an argument name to indicate that it won't be used.<br>Surrounding space missing in default value assignment.